README.md

```diff
@@ -171,6 +171,17 @@ wrap(MyComponent)
   .mount()
 ```
 
+By default it will use the native javascript history API, but you can provide a method to be called for change the app route with [`changeRoute`](#changeRoute):
+```js
+import { configure } from 'wrapito'
+import { history } from 'app.js'
+
+configure({
+  ..configuration,
+  changeRoute: (route) => history.push(route)
```

Comment:
In this case I think the usage of the arrow function make it easier to understand vs just passing `history.push`